crates/core/src/client/constants.rs

```diff
@@ -39,3 +39,22 @@ pub mod gas {
     /// As a result, the priority fee is set to 0.
     pub const MAX_PRIORITY_FEE_PER_GAS: U128 = U128::ZERO;
 }
+
+// This module contains constants which are being used in place of real data that should be fetched
+// in production.
+pub mod mock_evm_data {
```

Comment:
For reth types, such as `U256::from(...)` etc.
Can we try to do an implem with lazy_static! macro?

Then, the constants could be directly typed as U256 and not have to be derived later?

Reply:
Lemme investigate this one!